core/scripts/cli/sponsor/marketUtils.js

```diff
@@ -19,7 +19,12 @@ const getMarketSummary = async (web3, artifacts) => {
   for (const address of contractAddresses) {
     // The governor is always registered as a contract, but it isn't an ExpiringMultiParty.
     if (address !== Governor.address) {
-      emps.push(await ExpiringMultiParty.at(address));
+      // Additional check that the address is a contract.
+      try {
```

Comment:
@ptare I added this check, which I brought up offline as a potential bug. Before, if you ran the `RequestOraclePrice.js` script, it would register web3.accounts[0] as a "registered contract", but attempting to call `EMP.at(accounts[0])` would fail.

Reply:
This is probably also a good safety check to have here anyway, so LGTM.